examples/mpns.rb

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+
+$:.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
+
+require 'ruby-push-notifications'
+
+device_urls = [
+  'First device url here',
+  'Second device url here'
+]
+
+# Notification with toast type
+notification = RubyPushNotifications::MPNS::MPNSNotification.new device_urls, { title: 'Title', message: 'Hello MPNS World!', type: 'toast' }
```

Comment:
Line is too long. [141/80]<br>Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.